src/shared/dropdown-wrapper/dropdown-wrapper.styles.tsx

```diff
@@ -189,10 +189,17 @@ export const Divider = styled.div`
     margin: 0 0.5rem;
 `;
 
-export const LabelContainer = styled.div`
+export const LabelContainer = styled.div<{ disabled?: boolean }>`
```

Comment:
non-html attributes should be prefixed with `$` to mark them as [transient](https://styled-components.com/docs/api#transient-props)

```suggestion
export const LabelContainer = styled.div<{ $disabled?: boolean }>`
```

Reply:
done and shifted into style prop